tox.ini

```diff
@@ -27,6 +28,7 @@ deps =
     django32: Django==3.2.*
     django40: Django==4.0.*
     django41: Django==4.1.*
+    django42: Django==4.2*
```

Comment:
```suggestion
    django42: Django==4.2.*
```

Reply:
It wasn't a typo because at the point in time, there was only 4.2rc1 no 4.2. So if I aadd the `.` the whole thing will fail.

Reply:
I see that you have already updated the tox.ini, so I created PR #3319 to update other stuff to make it compliant with the support of 4.2